### ISSUE TITLE
Review Design Wireframe: Privacy Policy page

### ISSUE THREAD

**Issue by lem31** [NoroffFEU/musikkforandrerliv.no#103]
- [x] Visual Hierarchy
- [x] Layout
- [x] Consistency
- [x] Screen size
- [x] Usability Principles
- [x] Web Content Accessibility Guidelines (WCAG)
- [x] Colors
- [x] Fonts
- [x] Whitespace and padding
- [x] Image quality
- [x] Iconography
- [x] Micro interactions
- [x] Error states and feedback
- [x] Performance Optimization
- [x] Design documentation

See Design wireframe/prototype checklist for more info

**Comment by Oyvindeavor** [comment]
## Desktop
![Desktop Screenshot](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/133252875/0d9b3c32-0579-451d-b164-af40d70daf84)
- The "More" button has two arrows and does not seem to work.

### Text under governing law looks too long compared to the rest of the texts
![Text under governing law](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/133252875/da1396dc-6ab7-4398-b03e-6d7c844f6796)

### News button not working in the footer
![News button footer](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/133252875/2ef719b7-b21d-4c1e-8ac0-e396316bea1b)

### Follow us buttons not working in the footer
![Follow us footer](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/133252875/954ddca2-43ad-4123-8d4e-6ece38fe6d70)

### News, search buttons not working in the header
![Header buttons](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/133252875/54828fbf-32c8-49bb-90e2-079f31beedb6)

### Inconsistent margin between last dropdown and governing law compared to the rest
![Inconsistent margin](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/133252875/5efa30ea-428f-428a-914c-fb2c3694ecb0)
![Inconsistent margin continued](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/133252875/f2b5bea5-ee22-4afe-97bb-62fb5c2d445b)

### Wrong font for text under our privacy policy and under governing law
- Text used is Inter, while it should be Montserrat according to the typography guidelines.
![Wrong font](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/133252875/c38f5991-9995-4c48-8664-56aac955c523)

## Mobile
- On mobile, the text under our privacy policy looks a bit weird and should be aligned left for consistency along with the rest of the design.
![Mobile privacy policy text](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/133252875/0ee11a9a-17f2-4215-9649-a42be86350aa)

### Wrong font used under our privacy policy and under governing law
- Font used is Inter, while it should be Montserrat according to the typography guidelines.
![Wrong font mobile](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/133252875/ac5cd6bb-f2fd-4b63-8437-abf10b5171da)

### News button in the footer not working
![Mobile news button](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/133252875/8e234f70-1634-47d3-8e3d-8deabf8d2563)

### Donate button in the footer not working
![Mobile donate button](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/133252875/82bd8f57-de82-44b6-b08f-e5e34fdedf8c)

### Terms and conditions in the footer not working
![Mobile terms and conditions](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/133252875/465342f9-0277-4774-a36f-05912a312c24)

### Hamburger menu is not working
![Mobile hamburger menu](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/133252875/6c1e7192-e246-4de3-ac6d-66c9a40de209)

### Follow us
- Follow us social media icons not working.
![Follow us mobile](https://github.com/NoroffFEU/musikkforandrerliv.no/assets/133252875/d559005d-e39a-4c06-b71f-e9e9915e67c0)

**Comment by Vickabu** [comment]
Font updated, text for mobile changed to "aligned left" and chenged the margin between last dropdown and governing law. No changes regarding prototyping(button functions) since that is not a part of the design this time.

**Comment by Oyvindeavor** [comment]
Closing as it is fixed.
